projects/ng-select2-component/src/lib/select2.component.html

```diff
@@ -74,7 +74,12 @@
                        spellcheck="false"
                        [attr.tabindex]="this.isOpen ? tabIndex : '-1'">
             </div>
-            <div class="select2-results">
+            <div class="select2-results"
+                 infiniteScroll
```

Comment:
It's possible to make it optional?

Reply:
Absolutely! I added a flag for infiniteScrollDisabled set to true, this means that only if I manually set that input to false, it should work.